test/runIntegrationTests.ts

```diff
@@ -24,19 +24,18 @@ export async function runIntegrationTests(projectName: string) {
         verbose: true,
     } as Config.Argv;
 
-    let filter: string;
     if (process.env.TEST_FILE_FILTER) {
-        // If we have just a file, run that with runTestsByPath.
-        jestConfig.runTestsByPath = true;
+        // If we have just a file, run that with an explicit match.
```

Comment:
None of the changes here are explicitly related to the test reporting.  But while trying to get that to work I fixed the issue where the launch.json threw out a ton of errors even when the tests passed.

Now looks like
<img width="636" alt="image" src="https://github.com/dotnet/vscode-csharp/assets/5749229/41034a29-fe13-4943-bedf-50a3bf2c1f7a">